plugins/inputs/windows_event_log/wineventlog/wineventlog.go

```diff
@@ -171,7 +159,8 @@ func (l *windowsEventLog) Open() error {
 	// Subscribe for events
 	eventHandle, err := EvtSubscribe(0, uintptr(signalEvent), channelPath, query, bookmark, 0, 0, EvtSubscribeStartAfterBookmark)
 	if err != nil {
-		fmt.Errorf("error when subscribing for events. Details: %v", err)
+		log.Printf("EvtSubscribe(), name %v, err %v", l.name, err)
+		return err
```

Comment:
This bug was found by the new unit test for the `Open()` function.

Reply:
In `windows_event_log.go` the `Start()` method for this plugin will fail if any of the monitored event logs fails `Init()`.

For new users, if they configure the CWA to monitor 5 event logs and 1 of them is invalid, I think it is better to fail like this so the user is more likely to notice and fix the configuration.

Although existing users may already have _bad_ event logs listed, and this would be a breaking change in behavior. So I will undo this change.

Reply:
Ah great point that this would be a breaking change.  That's unfortunate but I think you're right that we can't change it now.